roles/redpanda_broker/Makefile

```diff
@@ -1,22 +1,27 @@
-.PHONY: test reqs do
-
 PYTHON ?= python
 PIP ?= pip
 
 DIR := tests
+### TODO revert to original scripts
```

Comment:
Is this a leftover?

Reply:
Yeah, leftover from me excluding the other tests I'll clean it up